pkgs/applications/system/linux-enable-ir-emitter/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, makeWrapper
+, fetchFromGitHub
+, fetchurl
+, meson
+, ninja
+, pkg-config
+, python3
+, opencv
+, usbutils
+}:
+stdenv.mkDerivation rec {
+  pname = "linux-enable-ir-emitter";
+  version = "4.5.0";
+
+  src = fetchFromGitHub {
+    owner = "EmixamPP";
+    repo = pname;
+    rev = version;
+    hash = "sha256-Dv1ukn2TkXfBk1vc+6Uq7tw8WwCAfIcKl13BoOifz+Q=";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config makeWrapper ];
+  buildInputs = [ python3 opencv ];
+
+  patches = [ ./remove-boot-set.patch ];
```

Comment:
By convention, patches go after `src` so it is harder to forget to update them, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.